stable/filebeat/values.yaml

```diff
@@ -40,6 +40,23 @@ config:
     rotate_every_kb: 10000
     number_of_files: 5
 
+  http.enabled: false
```

Comment:
Is http.enabled a filebeat configuration?  If not I would separate it out into it's own thing.  Also the dot in the string may cause issues with people trying to set it on the command line as I believe it will be treated as a map.

Reply:
Just looked at the docs, I see this is a filebeat configuration.  LGTM

Reply:
Agreed, having a period in the key makes it a bit tricky to use with `--set` and the usage of the key in the template is not the most intuitive. We could try to add a separate, more helm-friendly key value pair that enables the containerPort but I'm not sure how we could conditionally merge a yaml block into the filebeat.yml Secret as its setup now. Any ideas?

Reply:
I would be fine with keeping it as it is, but I would add a comment around it saying that it can't be set on the command line.